care/facility/api/serializers/patient.py

```diff
@@ -465,10 +470,15 @@ def validate_empty_values(self, data):
     class Meta:
         model = PatientNotes
         fields = (
+            "id",
```

Comment:
Remember to return external_id in this key

Reply:
It returns the object's pk, Is there a reason to prefer external_id over pk?

Reply:
yes we only expose external_id as id

Reply:
Done.